### DISCUSSION TITLE
[Feature] Full size viewer in duplicate review

### DISCUSSION THREAD

**Discussion by bathtubfulloftoast** [immich-app/immich#10173] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

allow duplicate images and videos to be reviewed in full

the current issue is that i have seen  two videos with the same thumbnail and i cannot review if they are actually the same

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by chrisscotland** [comment]
Yes please this would be very helpful for review of images. Some of my duplicates appear to be edited versions with subtle changes in white balance/colour that are hard to spot in the thumbnail.

**Reply by blublub** [reply]
yes, very helpful!

I would also like to add a that 2 images can be shown next to each other scales to full the screen and if more than 2 duplicates are found that one can skip through them in that mode.

**Comment by Hantick** [comment]
We could provide a button to just see a full-size photo
![image](https://github.com/immich-app/immich/assets/36815427/705c5b9f-8818-4d6c-8435-e928895aba0b)

**Reply by alextran1502** [reply]
Implemented and will be available in the next release
